src/release.js

```diff
@@ -236,27 +236,32 @@ export default function taoExtensionReleaseFactory(params = {}) {
         async createPullRequest() {
             log.doing('Create the pull request');
 
-            const pullRequest = await githubClient.createReleasePR(
-                data.releasingBranch,
-                params.releaseBranch,
-                data.version,
-                data.lastVersion,
-                subjectType,
-            );
-            if (pullRequest && pullRequest.state === 'open') {
-                data.pr = {
-                    url: pullRequest.html_url,
-                    apiUrl: pullRequest.url,
-                    number: pullRequest.number,
-                    id: pullRequest.id,
-                    full_name: pullRequest.head.repo.full_name,
-                };
-                const labels = ['releases'];
-                await githubClient.addLabel(data.pr.full_name, data.pr.number, labels);
-                log.info(`${data.pr.url} created`);
-                log.done();
-            } else {
-                log.exit('Unable to create the release pull request');
+            try {
+                const pullRequest = await githubClient.createReleasePR(
+                    data.releasingBranch,
+                    params.releaseBranch,
+                    data.version,
+                    data.lastVersion,
+                    subjectType,
+                );
+                if (pullRequest && pullRequest.state === 'open') {
+                    data.pr = {
+                        url: pullRequest.html_url,
+                        apiUrl: pullRequest.url,
+                        number: pullRequest.number,
+                        id: pullRequest.id,
+                        full_name: pullRequest.head.repo.full_name,
+                    };
+                    const labels = ['releases'];
+                    await githubClient.addLabel(data.pr.full_name, data.pr.number, labels);
+                    log.info(`${data.pr.url} created`);
+                    log.done();
+                } else {
+                    log.exit('Unable to create the release pull request');
+                }
+            } catch (err) {
```

Comment:
If `err` from githubClient is not providing any helpful info, may we can log some state from the program? For example `data`. It's your call.

Reply:
I think logging the data could be a good improvement, or at least the url / apiurl and id, for logging purposes it should do fine.

The info that arrives over here are the one coming straight from this line in octonode: https://github.com/pksunkara/octonode/blob/644375ea24c8b766cd6b3b2035de85665e3f9d51/src/octonode/repo.coffee#L483

Reply:
In the latest change, you no longer log `err`.
I think it's worthwhile to keep `log.error(err)`, as that was the point of catching it.

Reply:
Indeed, but the err only contains that basic string. I'll keep both changes then!

Reply:
@oatymart fixed, I think :)